src/components/AssignmentForm/AssignmentForm.js

```diff
@@ -99,7 +99,7 @@ export default class AssignmentForm extends Component {
           //   <textarea placeholder="PA Guide" rows="6" className="form-control" />
           // </div>
         }
-        <button className={(loading ? 'disabled ' : '') + 'btn btn-block btn-primary'}>
+        <button disabled={!(this.state.name && this.state.input)} className={(loading ? 'disabled ' : '') + 'btn btn-block btn-primary'}>
```

Comment:
I prefer to be under 100 column width. You should go on sublime and add the 100 column width ruler in User Preferences. If it goes over the 100 column width you should separate each property to its own property. Also notice how at the beginning of my methods I grab the input already so you should reuse that.